tests/test_url_config.py

```diff
@@ -117,12 +117,6 @@ def test_leading_zero_int(self):
         self.assertEqual(p.test_config, "0666")
         self.assertIsInstance(p.test_config, str)
 
-    @unittest.skipIf(not straxen.utilix_is_configured(), "No db access, cannot test CMT.")
```

Comment:
Shall we add a test for xedocs protocol like what @LuisSanchez25 did [here](https://github.com/XENONnT/straxen/pull/1124/files)?

Reply:
xedocs protocol is almost tested everywhere in straxen after https://github.com/XENONnT/straxen/pull/1235